kb/pattern/scs/class.scs

```diff
@@ -0,0 +1,13 @@
+concept_your_class_identifier
+<- sc_node_class;
+<- class;
+<= nrel_inclusion: 
+	concept_superclass_identifier;
+=> nrel_inclusion:
+	concept_subclass_identifier;
+=> nrel_main_idtf: 
+	[идентификатор] (*<- lang_ru;;*);
```

Comment:
```suggestion
	[идентификатор] (* <- lang_ru;; *);
```
It would be greate to fix formatting to make this patterns easy to read and understand. Add spaces with `*` Change everywhere, please.